### DISCUSSION TITLE
How do you remove the [world] tag from the chat line?

### DISCUSSION THREAD

**Discussion by MetaBeam** [TownyAdvanced/Towny#7644] [Q&A] [answered]
I'm tryna remove the [world] tag but idk how

**Comment by LlmDl** [comment, marked as answer]
In order to edit your chat line you need to open your towny\settings\chatconfig.yml file. Once you have it open you can navigate to the channel_formats lines, find the global one. Then begin to cross reference the tags used, with the tags in the chatconfig.yml's comments section.

Once you've customized the global channel_format line to your liking save the chatconfig.yml and use /townychat reload ingame.
